lumicks/pylake/detail/bead_cropping.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+
+
+def find_beads(
+    summed_kymo, bead_diameter_pixels, plot=False, percentile=70, allow_negative_beads=True
+):
+    """Find bead edges in kymograph image sum
+
+    Attempts to search for the bead edges by checking where the fluorescence drops below a
+    threshold. Only intended for non-moving beads.
+
+    Parameters
+    ----------
+    data : np.ndarray
+        Kymograph image data (single channel summed along the time axis).
+    bead_diameter_pixels : float
+        Estimate for the bead size in pixels.
+    plot : bool
+        Plot result
+    percentile : int
```

Comment:
maybe something like `threshold_percentile` would be more descriptive?

Reply:
Works for me!